modules/p2/ding_serendipity_ting_entity/ding_serendipity_ting_entity.module

```diff
@@ -221,14 +229,14 @@ function ding_serendipity_ting_entity_ting_object_adhl_serendipity_add($context)
 
   $type = strtolower($obj->getType());
   if (!empty($obj) && isset($accepted_types[$type]) && $accepted_types[$type]) {
-    $record = $obj->getRecord();
 
     if ($obj->getIsbn()) {
       if (($agency = variable_get('ting_agency', '')) != '') {
         $agency .= ':';
       }
 
-      $recommendations = ting_get_object_recommendations($obj->getIsbn());
+      // TODO BBS-SAL BBS-213: Implement search provider call here?
```

Comment:
Same as the others :-)

Reply:
A ticket should be created for this.

We have not had the time to migrate this module in time. The current situation will not harm OpenSearch libraries.

Reply:
Please disregard the comment above. This has to do with the decision about how ADHL should be handled going forward.